src/main/java/org/scalameter/annotations/benchmark.java

```diff
@@ -0,0 +1,31 @@
+package org.scalameter.annotations;
```

Comment:
We should avoid using plural in package names (despite earlier violations) - let's make this `annotation`.